client/transaction-pool/graph/src/validated_pool.rs

```diff
@@ -90,9 +90,25 @@ pub type ValidatedTransactionFor<B> = ValidatedTransaction<
 	<B as ChainApi>::Error,
 >;
 
+/// A closure that returns true if the local node can author blocks.
+pub struct CanAuthor(Box<dyn Fn() -> bool + Send + Sync + 'static>);
```

Comment:
```suggestion
pub struct CanAuthor(Box<dyn Fn() -> bool + Send + Sync>);
```
`'static'` is implied from the `Box`, no?